UofS-Pulse-Binfo/kp_frontpage
# kp frontpage fresh layout for review

This PR will update KP Frontpage with new and fresh frontend.
To test:
1. Uninstall current KP Frontpage from your Tripal site instance. This will ensure that unused files/configuration/relations from the old layout will be deleted.
2. Update KP Frontpage and enable module from modules panel. At this stage the new module will create fields required in News and Events sections as well as initialize materialized view for a quick summary count of data under data section.
3. Congratulations! you now replaced the old KP with this new layout. You might want to add entries to news and events section.

NOTE: 
- Data count for genotype calls takes forever to generate, I added a limit in the query to only count the first 10 records. @laceysanderson you might have a swift method.
- The real-life image of each crops are result of google. I will do some follow up on our breeders to send us couple of pictures. So far I already have picture of Lentil (from Derek), Chickpea (from Bunyamin), Faba (from Lacey). Awaiting pictures of Dry Bean (from Kirstin) and Pea (from Tom).
